src/table/settings/game_settings.ini

```diff
@@ -319,6 +319,19 @@ strhelp  = STR_CONFIG_SETTING_LIMIT_TRAIN_ACCELERATION_HELPTEXT
 cat      = SC_EXPERT
 patxname = ""realistic_braking.vehicle.limit_train_acceleration""
 
+[SDT_VAR]
+var      = vehicle.train_acc_braking_percent
+type     = SLE_UINT8
+flags    = SF_PATCH
+def      = 100
+min      = 5
+max      = 200
+str      = STR_CONFIG_SETTING_TRAIN_ACC_BRAKING_PERCENT
+strhelp  = STR_CONFIG_SETTING_TRAIN_ACC_BRAKING_PERCENT_HELPTEXT
+strval   = STR_CONFIG_SETTING_TRAIN_ACC_BRAKING_PERCENT_VALUE
+cat      = SC_EXPERT
+patxname = ""realistic_braking.vehicle.train_acc_braking_percent""
```

Comment:
The patxname field isn't needed for new settings, it's to do with previous savegame formats so can be left out here.